examples/demos/boot_from_SAN_and_server_profile_and_volume_migration/README.md

```diff
@@ -0,0 +1,17 @@
+# Demo
```

Comment:
Also, name of the folder seems weird to me.
Perhaps `boot_from_SAN+server_profile_and_volume_migration`?
`boot_from_SAN&volume_migration`?

Not really enjoying any a lot, @tmiotto any bright ideas here?

Reply:
Maybe `boot_from_san_and_migration` There is no need to be that level of specific in the title.